src/styles/app/50-flow.css

```
@@ -0,0 +1,303 @@
+	color: var(--text-secondary);
+	font: inherit;
+	font-size: 12px;
+	line-height: 1.45;
+}
+
+.flowNodeMuted {
+	margin: 0;
+	color: var(--text-tertiary);
+	font-size: 12px;
+	line-height: 1.4;
+	word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)